internal/services/compute/linux_virtual_machine_resource_disk_os_test.go

```diff
@@ -141,13 +141,29 @@ func TestAccLinuxVirtualMachine_diskOSDiskEncryptionSetUpdate(t *testing.T) {
 	})
 }
 
-func TestAccLinuxVirtualMachine_diskOSEphemeral(t *testing.T) {
+func TestAccLinuxVirtualMachine_diskOSEphemeralDefault(t *testing.T) {
 	data := acceptance.BuildTestData(t, "azurerm_linux_virtual_machine", "test")
 	r := LinuxVirtualMachineResource{}
 
 	data.ResourceTest(t, r, []acceptance.TestStep{
 		{
-			Config: r.diskOSEphemeral(data),
+			Config: r.diskOSEphemeralDefault(data),
+			Check: acceptance.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+				check.That(data.ResourceName).Key("os_disk.0.diff_disk_settings.0.placement").HasValue("CacheDisk"),
```

Comment:
This can be removed as all the properties will be checked in the import step below (even for absent optional properties).